### ISSUE TITLE
Release 2.76

### ISSUE THREAD

**Issue by gamontoya** [ucsdlib/damsmanager#385]
### Descriptive summary

Descriptive summary
Version: 2.76
Release Manager: @lsitu
Product Owner and Reviewers: @gamontoya
Operations: @rstanonik

------------
### To Verify
[ x ] - [Batch metadata overlay fails on validated metadata, due to FAST topics](https://github.com/ucsdlib/damsmanager/issues/378)
[ x ] - [Contextual display for Cite this Work enhancement](https://github.com/ucsdlib/damspas/issues/732)
[ ] - [CIL metadata transformation QA: Rename duplicate content files](https://github.com/ucsdlib/damsmanager/issues/317#issuecomment-573093895)
[ x ] - [DAMS Collection Status Report not working](https://github.com/ucsdlib/damsmanager/issues/382)
[ ] - [CIL Harvesting - update to notification email addresses](https://github.com/ucsdlib/damsmanager/issues/383)

**Comment by gamontoya** [comment]
@lsitu Please reference tickets to verify above.

**Comment by lsitu** [comment]
@gamontoya @arwenhutt  @abbypenn93 @remerjohnson DAMS Manager release 2.76 is ready on staging for review now. Thanks.

**Comment by arwenhutt** [comment]
@remerjohnson & @abbypenn93 
I've added names for leads on testing each ticket in this release.  Let us all know if you need assistance or replication for testing.  Let me know if you have any questions, thanks!

[ ] - Batch metadata overlay fails on validated metadata, due to FAST topics @remerjohnson 
[ ] - Contextual display for Cite this Work enhancement @abbypenn93 
[ ] - CIL metadata transformation QA: Rename duplicate content files @abbypenn93 
[ ] - DAMS Collection Status Report not working @arwenhutt 
[ ] - CIL Harvesting - update to notification email addresses @abbypenn93

**Comment by arwenhutt** [comment]
@lsitu I think you said (in #382) that the Collection Status Report would need to be tested on production.  I did try running the report on staging just in case, and it worked - but will test again once it's deployed to production.

**Comment by lsitu** [comment]
@arwenhutt Yes, we need to test it on prod as well. Thanks.

**Comment by arwenhutt** [comment]
@abbypenn93 the collection report bug was super quick to check so I did a little testing for https://github.com/ucsdlib/damspas/issues/732 (results added to that ticket)

**Comment by arwenhutt** [comment]
@gamontoya @lsitu 
A testing update.  We've completed testing for the three tickets that we can evaluate now and they look good.  We can test the other two once the CIL harvest currently running is done.

**Comment by gamontoya** [comment]
@arwenhutt Thanks for the update.

**Comment by lsitu** [comment]
@arwenhutt @gamontoya I am tracking the process and I's still running at this time. It looks like downloading of those giant zip files are very slow.

**Comment by arwenhutt** [comment]
@lsitu @gamontoya Just wondering if it's possible to split this into two releases, one for the non-cil code changes (that have already been tested) and another for the cil changes?  If so, we could deploy the non-cil changes to production right away, rather than having to wait for the harvest process to finish and be evaluated.

It'd be great to have the batch overlay tool usable again, and get started on fixing the items that have lost their unit metadata since this impacts the UI and user access.

**Comment by gamontoya** [comment]
@lsitu What do you think? ⬆️ ?

**Comment by lsitu** [comment]
@gamontoya @arwenhutt I think we can deploy the release to prod as a whole if you want. If we found that we need some changes for CIL Harvesting after QA-ed, we can create another PR for it.

**Comment by arwenhutt** [comment]
@lsitu That seems pretty safe to me, since CIL isn't fully in production tool yet, what do you think @gamontoya ?

**Comment by gamontoya** [comment]
@lsitu @arwenhutt Yes, let's do it. 

@lsitu Go ahead and ask Ops to deploy to prod.

**Comment by lsitu** [comment]
@rstanonik DAMS Manager release 2.76 is ready for prod now. But it has dependency on release private_config Release 2.2 https://github.com/ucsdlib/private_config/issues/23. So we need to deploy private_config Release 2.2 first. Thank you.

**Comment by arwenhutt** [comment]
@lsitu & @gamontoya thanks!

Once the private_config release is deployed, we'll test the non-cil tickets on production.  

One of our top priorities after that's done is to clean up the records effected by https://github.com/ucsdlib/damspas/issues/732  

For that, we'll need to identify what records are effected, so I created a report request ticket: https://github.com/ucsdlib/damsmanager/issues/389

**Comment by rstanonik** [comment]
Deployed to production this morning.
First I deployed private_config (damsmanager properties).
The tomcat was restarted.

**Comment by lsitu** [comment]
Thanks @rstanonik.

**Comment by arwenhutt** [comment]
@remerjohnson & @abbypenn93 the following issues can be tested (carefully) on production now!

[  ] - [Batch metadata overlay fails on validated metadata, due to FAST topics](https://github.com/ucsdlib/damsmanager/issues/378) @remerjohnson 
[  ] - [Contextual display for Cite this Work enhancement](https://github.com/ucsdlib/damspas/issues/732) @abbypenn93 
[  ] - [DAMS Collection Status Report not working](https://github.com/ucsdlib/damsmanager/issues/382) @arwenhutt

**Comment by lsitu** [comment]
@arwenhutt The CIL Harvest process was done over the weekend. Did all members in the Digital_Object_Metadata_Unit@ucsd.edu group receive email notification?

[ ] - [CIL Harvesting - update to notification email addresses](https://github.com/ucsdlib/damsmanager/issues/383)

**Comment by arwenhutt** [comment]
@lsitu ack, no.  I just looked and I think it's my bad, I left the "ad" off the email address -- it should have been "Digital_Object_Metadata_Unit@AD.UCSD.EDU"
sorry!  I'll add this to #383 (this isn't urgent to fix)

**Comment by lsitu** [comment]
@arwenhutt I would check with OPs to see what we can do. @rstanonik Could we make Digital_Object_Metadata_Unit@AD.UCSD.EDU an alias to Digital_Object_Metadata_Unit@ucsd.edu? Thanks.

**Comment by rstanonik** [comment]
I'll ask hostmaster@ucsd.edu

**Comment by lsitu** [comment]
Thanks @rstanonik.

**Comment by remerjohnson** [comment]
@arwenhutt The test on production for #378 went well

**Comment by arwenhutt** [comment]
@lsitu & @rstanonik thanks!

**Comment by rstanonik** [comment]
Campus says the alias will be ready by 11am today.

**Comment by lsitu** [comment]
That sound great! Thanks @rstanonik .

**Comment by gamontoya** [comment]
Kamino cloned this issue to [ucsdlib/damsmanager](https://github.com/ucsdlib/damsmanager/issues/399)
